src/compile/axis/encode.ts

```diff
@@ -15,8 +15,8 @@ export function labels(model: UnitModel, channel: PositionScaleChannel, specifie
 
   let labelsSpec: any = {};
 
-  // Text
-  if (isTimeFieldDef(fieldDef)) {
+  // We use a label encoding instead of setting the `format` property because Vega does not let us determine how the format should be interpreted.
+  if (isTimeFormatFieldDef(fieldDef)) {
     const isUTCScale = model.getScaleComponent(channel).get('type') === ScaleType.UTC;
 
     const expr = timeFormatExpression(
```

Comment:
My hope was that we could completely delete this function actually. But I guess it's an unrelated change so let's do this in a follow up.

Reply:
Never mind.